File: Redis_PoC/php/code/sample2.php
<?php
require "vendor/autoload.php";
Predis\Autoloader::register();

$redis_ip = "127.0.0.1";
if (getenv("REDIS_SERVER_ADDRESS"))
  $redis_ip = getenv("REDIS_SERVER_ADDRESS");

$redis_port = 6379;
if (getenv("REDIS_SERVER_PORT"))
  $redis_port = getenv("REDIS_SERVER_PORT");

$redis = new Predis\Client(array('host' => $redis_ip, 'port' => $redis_port));


$pipe = $redis->pipeline();
$pipe->set('Number',123)->incrby('Number',123)->execute();
print "Number = " . $redis->get('Number') . "\n";

?>

File: Redis_PoC/python/code/sample2.py
import redis
import time
import os

redis_ip = os.environ.get("REDIS_SERVER_ADDRESS","127.0.0.1")
redis_port = os.environ.get("REDIS_SERVER_PORT",6379)

r = redis.StrictRedis(host= redis_ip, port=redis_port)

# simple set,increase
pipe = r.pipeline()
pipe.set('Number',123).incr('Number',123).execute()
print "Number = %s" % r.get('Number')
